lib/cmd/common/admin-storeitems.js

```diff
@@ -189,7 +191,17 @@ function addIcon(guid, iconFile, cb) {
 }
 
 function addBinary(guid, type, binary, cb) {
-  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", binary, {"type":type,"guid":guid},  "application/octet-stream", cb);
+  var dataObject = [{
```

Comment:
This looks like it would be better in fhreq.uploadFile(), so that other file uploads with work with the proxy.  Is there a reason to not include it there instead?